### ISSUE TITLE
Clarification request: standardized "list" representation

### ISSUE THREAD

**Issue by jnpkrn** [ClusterLabs/fence-agents#275]
I'm just a messenger to deliver the necessity in standardizing upon the
(presumably machine processable) output from `list` actions, as discussed
in this pacemaker commit (therefore inviting @kgaillot to follow up here):

https://github.com/ClusterLabs/pacemaker/pull/1808/commits/60ad7730fbf34c1f67700bace39a083c0e3d1c31

By "standardizing", I'd imagine declaring the format explicitly in
https://github.com/ClusterLabs/fence-agents/blob/master/doc/FenceAgentAPI.md#agent-operations-and-return-values

Thanks

**Comment by kgaillot** [comment]
Agreed. For backward compatibility, I'd make it a recommendation than a requirement.

Separately but affecting the design here, we should come up with at least a recommendation on what characters are allowed in port names and aliases, possibly including escape sequences for special characters. Currently pacemaker only allows alphanumeric, dash, underbar, and dot, but there have been user requests for port names like "cs-rh7-1 [1/2]".
